src/extension/options-page/DashboardDialogs/Persona.tsx

```diff
@@ -81,11 +81,13 @@ export function PersonaCreateDialog() {
 export function PersonaCreatedDialog() {
     const { t } = useI18N()
     const { identifier } = useQueryParams(['identifier'])
-    const [persona, setPersona] = useState<Persona | null>(null)
-    useAsync(async () => {
-        if (identifier)
-            Services.Identity.queryPersona(Identifier.fromString(identifier, ECKeyIdentifier).unwrap()).then(setPersona)
+    const { error, value } = useAsync(async () => {
+        if (identifier) {
+            return Services.Identity.queryPersona(Identifier.fromString(identifier, ECKeyIdentifier).unwrap())
```

Comment:
refactor to multiple lines is good for reading

Reply:
Does this necessary to add another local var? I think it's readable for me. How about other guys?

Reply:
https://github.com/DimensionDev/Maskbook/pull/812#discussion_r385067542

Reply:
I think it's a personal favor. Can we add an eslint rule for this? If we can then I will refactor it.

Reply:
Let's customize an eslint config to be run as a part of the pre-commit hook

Reply:
The name of this rule could be `prefer-variable-parameter`. That makes the world harder. 😄

Reply:
it's not necessary to be a lint rule